spawn/runners/process_runner.py

```diff
@@ -63,7 +63,7 @@ def run(self):
         validate_file(self._input_file_path, 'input_file_path')
         validate_file(self._exe_path, 'exe_path')
         LOGGER.info('Executing \'%s\': %s', self._id, self.process_args)
-        output = subprocess.run(args=self.process_args, cwd=self._cwd,
+        output = subprocess.run(args=self.process_args, cwd=self._cwd, check=False,
```

Comment:
`check` is `False` by default

Reply:
yeah this is a pylint issue that you've already resolved on another branch